src/main/java/htsjdk/samtools/cram/ref/EnaRefService.java

```diff
@@ -11,12 +11,13 @@
 
 public class EnaRefService {
     private static final Log log = Log.getInstance(EnaRefService.class);
-    private static final int HTTP_OK = 200;
+    private static final int HTTP_OK = HttpURLConnection.HTTP_OK;
     private static final int HTTP_FOUND = 302;
-    private static final int HTTP_NOT_FOUND = 404;
-    private static final int HTTP_INTERNAL_SEVER_PROBLEM = 500;
+    private static final int HTTP_NOT_FOUND = HttpURLConnection.HTTP_NOT_FOUND;
+    private static final int HTTP_INTERNAL_SEVER_PROBLEM = HttpURLConnection.HTTP_INTERNAL_ERROR;
+    // this is a non-standard code and I'm not sure why it's here
     private static final int HTTP_CONNECTION_TIMEOUT = 522;
```

Comment:
This one is interesting (522), it is not a standard one... looking at the way is used with HTTP_INTERNAL_SVER_PROBLEM I wonder whether that case means to handle all the 5xx codes.

Is this one the reason why you prefer to redefined this constants in this class rather than use HttpURLConnection's directly?